src/utils/createDbConn.js

```diff
@@ -1,40 +1,59 @@
 const fs = require("fs");
 const { updateProjectDependencies } = require("./helper");
 const { render } = require("ejs");
-//function to create db service---------------------------------------------->
-function createDbConn(nodePath, dbName, defaultRoute,currentPath) {
-  if (dbName === "postgres" || dbName === "mysql") {
-    let package = { name: "sequelize", version: "^6.6.5" };
-    updateProjectDependencies(nodePath, package);
-    var fileName = "sequelize.js";
-    var modelName = "sequelizeModel.js";
-    if (dbName === "mysql") {
-      package = { name: "mysql2", version: "^2.3.0" };
-      updateProjectDependencies(nodePath, package);
-    } else {
-      package = { name: "pg", version: "^8.7.1" };
-      updateProjectDependencies(nodePath, package);
-    }
+//<----------------------------- Function to create db service -------------------------------------------->
+function createDbConn(nodePath, dbName, defaultRoute, currentPath) {
+  const dependencies = [];
+  let fileName;
+  let modelName;
+  const modelPath = nodePath + "/models";
+  const isPostgres = dbName === "postgres";
+  const isMySQL = dbName === "mysql";
+
+  if ( isPostgres || isMySQL) {
+    dependencies.push({ name: "sequelize", version: "^6.6.5" });
+    fileName = "sequelize.js";
+    modelName = "sequelizeModel.js";
+
+    isMySQL
+      ? dependencies.push({ name: "mysql2", version: "^2.3.0" })
+      : dependencies.push({ name: "pg", version: "^8.7.1" });
   } else {
-    let package = { name: "mongoose", version: "^6.0.2" };
-    updateProjectDependencies(nodePath, package);
-    var fileName = "mongoose.js";
-    var modelName = "mongooseModel.js";
+    dependencies.push({ name: "mongoose", version: "^6.0.2" });
+    fileName = "mongoose.js";
+    modelName = "mongooseModel.js";
   }
```

Comment:
might be better to use switch-case here, as we are having 3 or can be more DBs to handle & adding their dependencies
also to avoid adding mongoose in else case. (should added when selection is of mongoDB)